### ISSUE TITLE
roiManager ignores selection when saving

### ISSUE THREAD

**Issue by ATpoint** [imagej/ImageJ#249]
From my (probably wrong) understanding, this should save ROIs number 4 and 5 to RoiSet.zip, no? It saves the first two though.

MRE:

```java
newImage("Untitled", "16-bit black", 512, 512, 1);
makeRectangle(108, 51, 77, 54);
roiManager("Add");
makeRectangle(225, 181, 75, 56);
roiManager("Add");
makeRectangle(367, 118, 51, 54);
roiManager("Add");
makeRectangle(342, 263, 98, 69);
roiManager("Add");
makeRectangle(271, 367, 84, 72);
roiManager("Add");
roiManager("Select", newArray(3,4));
roiManager("Save selected", "C:/Users/atpoint/Desktop/RoiSet.zip");
```

Is this a bug or am I being stupid?

**Comment by ATpoint** [comment]
Any feedback? "It's a noob problem" is also a feedback I am fine with. Then I keep digging.

**Comment by rasband** [comment]
This bug is fixed in the ImageJ 1.54j22 daily build. The commit is at https://github.com/imagej/ImageJ/commit/6f6e828b3453fbfd59d4fd65731b5117e3d941d5.
